src/View/Admin/adminPlatrerie.html.twig

```diff
@@ -11,7 +11,7 @@
     <div class="row">
         <div class="col-sm-6 col-md-4 col-lg-3">
             <form action="?route=adminImageFondPlatrerie" method="post" enctype="multipart/form-data">
-                Select image to upload:
+                <label name="upload">Photo à modifier :</label>
```

Comment:
le label doit avoir un attribut for relié à l'id du champ, ça ne va pas sinon... à remodif partout, et c'est bon, merci